src/containers/LoginForm.js

```diff
@@ -24,9 +24,12 @@ export class LoginForm extends Component {
     event.preventDefault();
     fetchUserLogin(this.state.email, this.state.password)
       .then(data => {
-        const { updateUser, updateLoggedInStatus } = this.props;
-        updateUser({ ...data.user });
-        updateLoggedInStatus(true);
+        const { updateUser, updateLoggedInStatus, addRatings } = this.props;
+        fetchRatings(data.user.id)
```

Comment:
@hfaerber I got an error for this, but I think it's a quick fix that we need to import fetchRatings in from apiCalls.

Reply:
Thanks Peerat!  I lost it on the merge conflict but just added it back with that import corrected.  You may need to pull down again to see it.  Thank you!

Reply:
Got it, I see the fix!  Everything looks good to me.